spec/views/newspaper_works/base/_attribute_rows.html.erb_spec.rb

```diff
@@ -0,0 +1,30 @@
+# hyrax/spec/views/hyrax/base/_attribute_rows.html.erb_spec.rb
```

Comment:
I like having this spec, but suggest we change or augment the work attributes to include some (don't need all) of the Newspaper fields. Could use `publication_date`, `place_of_publication`, `issn`, etc. Then need to test for presence of these fields on `page`.